docs/clusterdefinition.md

```diff
@@ -30,36 +30,38 @@ Here are the valid values for the orchestrator types:
+| serviceCidr                     | no       | IP range for Service IPs, Default is "10.0.0.0/16". This range is never routed outside of a node so does not need to lie within clusterSubnet or the VNET                                                                                                                                                                                                                                                                                                            |
+| useInstanceMetadata             | no       | Use the Azure cloudprovider instance metadata service for appropriate resource discovery operations. Default is `true`                                                                                                                                                                                                                                                                                                                                               |
+| useManagedIdentity              | no       | Includes and uses MSI identities for all interactions with the Azure Resource Manager (ARM) API. Instead of using a static service principal written to /etc/kubernetes/azure.json, Kubernetes will use a dynamic, time-limited token fetched from the MSI extension running on master and agent nodes. This support is currently alpha and requires Kubernetes v1.9.1 or newer. (boolean - default == false)                                                        |
+| loadBalancerSku                 | no       | Sku of Load Balancer and Public IP. Candidate values are: `basic` and `standard`. If not set, it will be default to basic. Requires Kubernetes 1.11 or newer.                                                                                                                                                                                                                                                                                                            |
+| excludeMasterFromStandardLB     | no       | Excludes master nodes from standard load balancer. Default is `true`. Requires Kubernetes 1.11 or newer.                                                                                                                                                                                                                                                                                                                                                             |

```

Comment:
Same thing here, which part requires 1.11? If the default is always true then what happens if the user is on k8s 1.8?